projects/new/seeds/questionnaires/new.rb

```diff
@@ -1,11 +1,11 @@
 # frozen_string_literal: true
 
-# db_title = 'Demo vragenlijst' # Dagboekvragenlijst moet geen titel hebben alleen een logo
+# db_title = '' # Shown to users as a big text, but cannot be localized, so often left empty.
```

Comment:
this is all commented, can it be deleted just like the other seeds?

Reply:
So this is the "new" project. Which is when you call the `deployment:create_project` rake task, it essentially copies this whole directory so that you have something to start from with some example files. 

Typically you would delete these files and put your own instead, but we also use it as like a hint for new projects so they know what to think about when making seeds (or to remind ourselves). For this reason, I'd like to leave the commenting in the seeds in the `new` project folder as verbose as possible.

Here I just changed the comments to English because they were in Dutch before, and I updated the syntax to reflect that for most questionnaires these days, the questionnaire key is the same as the filename.